components/elements/button.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <button
+    :class="{
+      primary,
+      secondary,
+      tertiary
+    }"
+  >
+    <slot />
+  </button>
+</template>
+
+<script>
+export default {
+  props: {
+    primary: {
```

Comment:
The class should be manage ba a single prop (Ex: `style`), unless a `button` can be `primary`, `secondary` and `tertiary` at the same time.

Reply:
i will not use a reserved word but yeah i agree even if i like to have boolean like this.